update-dependencies/update-dependencies.csproj

```diff
@@ -3,11 +3,13 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.0</TargetFramework>
+    <LangVersion>latest</LangVersion>
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.DotNet.VersionTools" Version="1.0.27-prerelease-01428-01" />
-    <PackageReference Include="System.Diagnostics.TextWriterTraceListener" Version="4.0.0" />
-    <PackageReference Include="NuGet.Packaging" Version="4.0.0" />
+    <PackageReference Include="Microsoft.DotNet.VersionTools" Version="2.2.0-prerelease-02431-01" />
+    <PackageReference Include="NuGet.Packaging" Version="4.3.0" />
```

Comment:
Can this package ref be removed? This is the version that `VersionTools` depends on (as of https://github.com/dotnet/buildtools/pull/1884).

Reply:
Removed.